src/components/MediaSettings/MediaDevicesSpeakerTest.vue

```diff
@@ -61,17 +69,17 @@ export default {
 
 	methods: {
 		t,
+
 		playTestSound() {
 			if (this.isPlayingTestSound) {
-				this.$store.dispatch('pauseWaitAudio')
+				this.soundsStore.pauseAudio('wait')
 				return
 			}
 			this.isPlayingTestSound = true
-			this.$store.dispatch('playWaitAudio').then((response) => {
-				response.addEventListener('ended', () => {
-					this.isPlayingTestSound = false
-				})
-			})
+			this.soundsStore.playAudio('wait')
+			this.soundsStore.audioObjects.wait.addEventListener('ended', () => {
+				this.isPlayingTestSound = false
+			}, { once: true })
```

Comment:
I wonder if we need error handling here in case playing doesn't work (in that case, we should not set the event listener)